README.md

```diff
@@ -202,7 +202,7 @@ If needed, you can specify the Scala version with `-PscalaVersion=2.13`.
     ./gradlew testJar
 
 ### Determining how transitive dependencies are added ###
```

Comment:
Maybe we should delete this and and move "Dependency Analysis" above "common build options"

Reply:
I agree. I missed that section. Let me update it.